pkg/stack/unwind/plan_table.go

```diff
@@ -110,21 +112,21 @@ func (ptb *PlanTableBuilder) PlanTableForPid(pid int) (PlanTable, error) {
 		}
 
 		abs := path.Join(fmt.Sprintf("/proc/%d/root", pid), m.File)
-		fdes, err := ptb.readFDEs(abs, m.Start)
+		fdes, err := ptb.ReadFDEs(abs, m.Start)
 		if err != nil {
 			level.Warn(ptb.logger).Log("msg", "failed to read frame description entries", "obj", abs, "err", err)
 			continue
 		}
 
-		res = append(res, buildTable(fdes))
+		res = append(res, BuildTable(fdes))
 	}
 
 	// TODO(kakkoyun): Merge and order instructions of PlanTables.
 	// TODO(kakkoyun): Figure out mapping start address usage.
 	return res[0], nil
 }
 
-func (ptb *PlanTableBuilder) readFDEs(path string, start uint64) (frame.FrameDescriptionEntries, error) {
+func (ptb *PlanTableBuilder) ReadFDEs(path string, start uint64) (frame.FrameDescriptionEntries, error) {
```

Comment:
Let's not export `ReadFDEs` and `BuildTable`.
We can add a `PrintTable` function to the package to the same thing.